README.md

```diff
@@ -7,26 +7,26 @@
 
-Latest release: **0.2.0**
-
 ## Description
 
-Thundernetes is a preview project from the [Azure PlayFab Multiplayer Servers (MPS)](https://docs.microsoft.com/en-us/gaming/playfab/features/multiplayer/servers/) team that enables you to run Linux game servers that use the [PlayFab Game Server SDK (GSDK)](https://github.com/PlayFab/gsdk) on your Kubernetes cluster. Thundernetes can be in the following scenarios:
+Thundernetes is a project from the [Azure PlayFab Multiplayer Servers (MPS)](https://docs.microsoft.com/en-us/gaming/playfab/features/multiplayer/servers/) team that enables you to run Linux game servers on your Kubernetes cluster. Thundernetes can be useful in the following scenarios:
 
 - host your game servers on a Kubernetes cluster, either on a public cloud provider or on-premises
 - do manual allocations of game server sessions
 - validate your game server integration with GSDK
 - as part of your iterative development process, you can use thundernetes to test your game server code before pushing it to the MPS service
 - as part of your CI/CD pipeline, you can publish the game server to a container registry and then have it deploy to a Kubernetes cluster where you can run your tests
 
-> We will refer to the Azure PlayFab Multiplayer Servers service as "MPS" in all pages of the documentation.
+For a game server to be able to run in thundernetes, it must use the [PlayFab Game Server SDK (GSDK)](https://github.com/PlayFab/gsdk) either directly on the game server binary or indirectly, via a wrapper (check [here](https://github.com/PlayFab/MpsSamples/tree/master/wrappingGsdk) for an example).
+
+> We will refer to the Azure PlayFab Multiplayer Servers as "MPS" in all pages of the documentation.
 
-Goal for thundernetes is to be portable with MPS - this means that your Linux Game Server that works on thundernetes should work with MPS.
+One of the main goals for thundernetes is to be portable with MPS - this means that your Linux Game Server that works on thundernetes will work with MPS and vice versa.
 
-Thundernetes requires a Kubernetes cluster with Public IP per Node. We've tested it extensively on [Azure Kubernetes Service - AKS](https://docs.microsoft.com/en-us/azure/aks/intro-kubernetes) as well as in local clusters using [kind](https://kind.sigs.k8s.io/). You also need to have ports 10000-50000 open in your cluster, since these are the ports that Thundernetes will use to receive traffic and forward to your game server.
+Thundernetes requires a Kubernetes cluster with Public IP per Node. We've tested it extensively on [Azure Kubernetes Service - AKS](https://docs.microsoft.com/en-us/azure/aks/intro-kubernetes) as well as in local clusters using [kind](https://kind.sigs.k8s.io/). You also need to have ports 10000-50000 open in your cluster, since these are the ports that Thundernetes will set up on your Kubernetes Nodes so they can receive game network traffic and forward to your game server Pod.

```

Comment:
Updated the link https://docs.microsoft.com/azure/aks/intro-kubernetes to remove en-us, so it will redirect to the user's home country/language.